server/stycoon/model_attack_command.go

```diff
@@ -10,31 +10,32 @@
 package stycoon
 
 type AttackCommand struct {
-	Command
+
+	Type string `json:"type"`
 
 	Target int64 `json:"target"`
+
+	Resource int64 `json:"resource,omitempty"`
```

Comment:
Why are you adding these fields? I don't see a change regarding Commands in swagger, so what caused this?

Reply:
good question, IDK